[{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "python-requestsexceptions",
	"version": "1.3.0-3",
	"section": "python",
	"priority": "extra",
	"architecture": "all",
	"depends": [
		"python-pbr (>= 2.0.0)",
		"python-urllib3 (>= 1.13)",
		"python:any (<< 2.8)",
		"python:any (>= 2.7.5-5~)"
	],
	"installedSize": 23,
	"maintainer": "Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>",
	"description": "import exceptions from bundled packages in requests. - Python 2.7",
	"homepage": "http://www.openstack.org/",
	"filename": "pool/main/p/python-requestsexceptions/python-requestsexceptions_1.3.0-3_all.deb",
	"size": 4044,
	"md5": "3a0844e56d74161d10089697439cbb56",
	"sha1": "d742bc2c4f637ecffefdcffa90258c74a29bdca5",
	"sha256": "4e8c3b6648d09ad902c7b697cd5db826e7f1b6dd89c25a19ec4c69729e763ca5",
	"descriptionMd5": "4d938c7960c4fdff8fd8bf6b6d2578cb"
}]